.github/stale.yml

```diff
@@ -0,0 +1,67 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 30
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels:
```

Comment:
Nits:
- "with _any_ of these labels"
- "are _checked_"

Reply:
Funny thing. You should probably raise a PR in the probot/stale repo since it was just copied from there: https://github.com/probot/stale :-D